apps/native-component-list/src/components/MonoTextWithCountdown.tsx

```diff
@@ -0,0 +1,127 @@
+import { Code } from '@expo/html-elements';
+import React, { PropsWithChildren, useCallback, useState, useRef, useEffect } from 'react';
+import { View, StyleSheet, TouchableOpacity, Text, Animated } from 'react-native';
+
+import Colors from '../constants/Colors';
+
+type Props = PropsWithChildren<{
+  /**
+   * Countdown timeout.
+   */
+  timeout?: number;
+
+  /**
+   * Called when the countdown ended or close button is pressed.
+   */
+  onCountdownEnded: () => void;
+}>;
+
+function MonoTextWithCountdown({ children, timeout = 8000, onCountdownEnded }: Props) {
```

Comment:
Could you explain the reason behind having the disappearing, pausable and resumable component? I don't see any benefit of this 😅

Reply:
When I was working with the old `ImagePicker` demo screen I was wondering why every result from the function call is presented differently (some additional section on the screen, alert dialog, etc.).
As almost every function in our APIs is returning some JSON-like object I started to wonder about the single unified way to present these results.
`MonoText` addresses this problem perfectly, because the results are clearly rendered on the screen.
But there's a problem with the `MonoText` though 😞 
It takes so much space on the screen even when the result could be rendered as a single line.
Most of the time results could be verified in just a few moments so I decided I should autohide `MonoText` by default, because it clutters the screen on the long run (especially if the screen presents more than one `FunctionDemo` - e.g. `ImagePicker` case).
But there're cases when you want to investigate the returned result more thoroughly and then autohiding after a fixed timeframe seems wrong, so I decided to add pause mechanism (resume mechanism was trivial at this point so I've added it as well) 😁
And that's the full story 😉